src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -249,6 +272,8 @@ $.BootstrapTable = class extends $.BootstrapTable {
       tmpItemIsExpected = value.toString().toLowerCase() === searchValue.toString().toLowerCase()
     } else if (column.filterStartsWithSearch) {
       tmpItemIsExpected = (`${value}`).toLowerCase().indexOf(searchValue) === 0
+    } else if (column.filterControl === 'datepicker') {
+      tmpItemIsExpected = new Date(value) === new Date(searchValue)
```

Comment:
I guess we need a date formatter.
E.g. in germany we never could use the datepicker, because we have a other format.